examples/scripts/custom_model.py

```diff
@@ -59,13 +59,13 @@
 model.submodels[
     "electrolyte conductivity"
 ] = pybamm.electrolyte_conductivity.LeadingOrder(model.param)
-model.submodels["negative sei"] = pybamm.sei.NoSEI(model.param, "Negative")
-model.submodels["positive sei"] = pybamm.sei.NoSEI(model.param, "Positive")
+model.submodels["negative sei"] = pybamm.sei.NoSEI(model.param)
+model.submodels["positive sei"] = pybamm.sei.NoSEI(model.param)
```

Comment:
can this be removed? looks like we don't need to set it in the lithium-ion models anymore?